commit 564f50c29fe242031844ecd49866800e3728cbe6
Author: ahmad1abbadi <156505848+ahmad1abbadi@users.noreply.github.com>
Date:   Wed Aug 21 02:22:11 2024 +0300

    fix installation some packages

diff --git a/installglibc.py b/installglibc.py
index c26f4a6..4601455 100644
--- a/installglibc.py
+++ b/installglibc.py
@@ -88,10 +88,8 @@ def termux_pkg():
     print(f"{R}[{W}-{R}]{G}{BOLD} This takes a few minutes it depends on your internet connection {W}")
     package_install_and_check("glibc-repo", "x11-repo")
     print(f"{R}[{W}-{R}]{G}{BOLD} glibc-repo + x11-repo installed {W}")
-    package_install_and_check("pulseaudio", "patchelf", "xkeyboard-config", "freetype", "fontconfig", "termux-x11-nightly", "termux-am zenity", "which", "alsa-lib-glibc", "bash", "curl", "sed", "cabextract")
-    print(f"{R}[{W}-{R}]{G}{BOLD}pulseaudio + termux-am +........... installed successfully {W}")
+    package_install_and_check("pulseaudio", "patchelf", "xkeyboard-config", "freetype", "fontconfig", "termux-x11-nightly", "termux-am", "zenity", "which", "alsa-lib-glibc", "bash", "curl", "sed", "cabextract")
     package_install_and_check("wget", "make", "libpng", "xorg-xrandr", "cmake", "unzip", "p7zip", "patchelf", "tur-repo", "tur-repo", "traceroute", "samba", "virglrenderer-android", "virglrenderer-mesa-zink", "zenity")
-    print(f"{R}[{W}-{R}]{G}{BOLD} patchelf + wget + make +........ installed successfully {W}")
     print("")
 def install_glibc_AZ():
     os.system("wget -q --show-progress https://github.com/ahmad1abbadi/darkos/releases/download/beta/glibc-darkos.tar.xz")
