commit bc04fcf2f96ca941a99a289d40173bfead7c8569
Author: ahmad1abbadi <156505848+ahmad1abbadi@users.noreply.github.com>
Date:   Mon Jul 15 21:12:51 2024 +0300

    Update darkos.py

diff --git a/darkos.py b/darkos.py
index 1d3e0bc..c678cba 100644
--- a/darkos.py
+++ b/darkos.py
@@ -17,7 +17,7 @@ C = "\033[1;36m"
 W = "\033[1;37m"
 BOLD = "\033[1m"
 
-current_version = "0.97"
+current_version = "0.971"
 url = 'https://raw.githubusercontent.com/ahmad1abbadi/darkos/main/currently%20version.txt'
 
 def extract_archive(file_path, extract_to):
