commit 3f213df850064a982824af35cbb656ab62fad7c0
Author: Ilya114 <93242944+Ilya114@users.noreply.github.com>
Date:   Tue Dec 31 18:22:22 2024 +0200

    Update box64droid.py

diff --git a/scripts/native/box64droid.py b/scripts/native/box64droid.py
index 5a5ad2e..aec6a70 100644
--- a/scripts/native/box64droid.py
+++ b/scripts/native/box64droid.py
@@ -28,7 +28,6 @@ def check_prefix():
     if not os.path.exists("/data/data/com.termux/files/home/.wine"):
         print("Wine prefix not found! Creating...")
         create_prefix()
-        main_menu()
 def recreate_prefix():
     prefix_path="/data/data/com.termux/files/home/.wine"
     os.system("clear")
