commit 24e5611620efbb133e42d4b26191bf2ef7b21768
Author: olegos2 <74909582+olegos2@users.noreply.github.com>
Date:   Sat Nov 23 11:35:32 2024 +0500

    No more tokens
---
 install | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/install b/install
index 8659def..cb0b878 100644
--- a/install
+++ b/install
@@ -66,7 +66,7 @@ esac
 echo "Installing mobox"
 
 function wget-git-q {
-    wget -q --retry-connrefused --tries=0 --header "PRIVATE-TOKEN: $PRIVATE_TOKEN" "https://gitlab.com/api/v4/projects/$PROJECT_ID/repository/files/$1/raw?ref=main" -O $2
+    wget -q --retry-connrefused --tries=0 "https://gitlab.com/api/v4/projects/$PROJECT_ID/repository/files/$1/raw?ref=main" -O $2
     return $?
 }
 
@@ -74,11 +74,9 @@ echo "Updating package manager"
 mkdir -p $PREFIX/glibc/opt/package-manager/installed
 
 if [ "$INSTALL_WOW64" = "1" ]; then
-echo "PRIVATE_TOKEN=glpat-h5r7HjKoPKZPxmfg79xs
-PROJECT_ID=54240888">$PREFIX/glibc/opt/package-manager/token
+echo "PROJECT_ID=54240888">$PREFIX/glibc/opt/package-manager/token
 else
-echo "PRIVATE_TOKEN=glpat-Xs4HfrCkMpbedkPycqzP
-PROJECT_ID=52465323">$PREFIX/glibc/opt/package-manager/token
+echo "PROJECT_ID=52465323">$PREFIX/glibc/opt/package-manager/token
 fi
 
 . $PREFIX/glibc/opt/package-manager/token
