diff --git a/app/build.gradle b/app/build.gradle
index eb83265..5566f42 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -97,14 +97,14 @@ android {
     compileSdk 34
     buildToolsVersion "35.0.0"
 
-    defaultConfig {
-        applicationId "com.winlator.cmod"
-        minSdkVersion 26
-        // We’re preserving Winlator’s request to keep targetSdkVersion at 28
-        targetSdkVersion 28
-
-        versionCode 20
-        versionName "7.1.4x-cmod"
+    defaultConfig {
+        applicationId "by.aero.so.benchmark"
+        minSdkVersion 26
+        // We’re preserving Winlator’s request to keep targetSdkVersion at 28
+        targetSdkVersion 28
+
+        versionCode 21
+        versionName "7.1.4x-benchmark-debug"
 
         // External native build from Winlator
         externalNativeBuild {
diff --git a/app/src/main/res/values/strings.xml b/app/src/main/res/values/strings.xml
index a940cf7..9750206 100644
--- a/app/src/main/res/values/strings.xml
+++ b/app/src/main/res/values/strings.xml
@@ -4,7 +4,7 @@
     <!ENTITY TERMUX_X11_APP_NAME "Winlator:X11">
     ]>
 <resources>
-    <string name="app_name">Winlator</string>
+    <string name="app_name">by.aero.so (benchmark)</string>
     <string name="run">Run</string>
     <string name="edit">Edit</string>
     <string name="remove">Remove</string>
@@ -635,4 +635,4 @@ E.g. <b>META</b> for <i>META key</i>, \n
     <string name="vkbasalt_sharpness_level">Sharpness Level</string>
     <string name="vkbasalt_sharpness_denoise">Sharpeness Denoise</string>
     <string name="shutdown">Shutdown</string>
-</resources>
\ No newline at end of file
+</resources>
