From e1668935c524431b1f54bf36eeac7b2129c74343 Mon Sep 17 00:00:00 2001
From: Codex <codex@local>
Date: Tue, 24 Feb 2026 03:01:40 +0300
Subject: [PATCH] winlator: bump Aero.so Winlator version to 0.2b

---
 app/build.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/build.gradle b/app/build.gradle
index 9d1e3d1..0df9275 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -109,7 +109,7 @@ android {
         targetSdkVersion 28
 
         versionCode 21
-        versionName "7.1.4x-benchmark-debug"
+        versionName "0.2b"
 
         // External native build from Winlator
         externalNativeBuild {
-- 
2.43.0

