From 9cecd63d2309475d7b030528ffb3e9468ed19ccf Mon Sep 17 00:00:00 2001
From: Codex <codex@local>
Date: Tue, 24 Feb 2026 02:08:55 +0300
Subject: [PATCH 2/2] bump app version to 0.2b

---
 app/build.gradle | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/app/build.gradle b/app/build.gradle
index f7e01b9..4cca117 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -108,8 +108,8 @@ android {
         // We’re preserving Winlator’s request to keep targetSdkVersion at 28
         targetSdkVersion 28
 
-        versionCode 20
-        versionName "7.1.4x-cmod-aeroso-benchmark"
+        versionCode 21
+        versionName "0.2b"
 
         // External native build from Winlator
         externalNativeBuild {
-- 
2.43.0
