commit f5eaf90d8683b30dd2e0110a6092126a2d150a2e
Author: ahmad1abbadi <156505848+ahmad1abbadi@users.noreply.github.com>
Date:   Mon Jul 15 23:29:14 2024 +0300

    Update installglibc.py

diff --git a/installglibc.py b/installglibc.py
index 309c92a..c26f4a6 100644
--- a/installglibc.py
+++ b/installglibc.py
@@ -90,7 +90,7 @@ def termux_pkg():
     print(f"{R}[{W}-{R}]{G}{BOLD} glibc-repo + x11-repo installed {W}")
     package_install_and_check("pulseaudio", "patchelf", "xkeyboard-config", "freetype", "fontconfig", "termux-x11-nightly", "termux-am zenity", "which", "alsa-lib-glibc", "bash", "curl", "sed", "cabextract")
     print(f"{R}[{W}-{R}]{G}{BOLD}pulseaudio + termux-am +........... installed successfully {W}")
-    package_install_and_check("wget", "make", "libpng", "xorg-xrandr", "cmake", "unzip", "p7zip", "patchelf", "tur-repo", "virglrenderer-android", "virglrenderer-mesa-zink", "zenity")
+    package_install_and_check("wget", "make", "libpng", "xorg-xrandr", "cmake", "unzip", "p7zip", "patchelf", "tur-repo", "tur-repo", "traceroute", "samba", "virglrenderer-android", "virglrenderer-mesa-zink", "zenity")
     print(f"{R}[{W}-{R}]{G}{BOLD} patchelf + wget + make +........ installed successfully {W}")
     print("")
 def install_glibc_AZ():
